proto/lbm/fswap/v1/fswap.proto

```diff
@@ -4,9 +4,43 @@ package lbm.fswap.v1;
 option go_package = "github.com/Finschia/finschia-sdk/x/fswap/types";
 
 import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+import "cosmos/bank/v1beta1/bank.proto";
 
-message Swapped {
+message Swap {
+  option (gogoproto.equal)            = true;
   option (gogoproto.goproto_stringer) = false;
-  string old_coin_amount              = 1 [(gogoproto.customtype) = "github.com/Finschia/finschia-sdk/types.Int", (gogoproto.nullable) = false];
-  string new_coin_amount              = 2 [(gogoproto.customtype) = "github.com/Finschia/finschia-sdk/types.Int", (gogoproto.nullable) = false];
+
+  string from_denom              = 1;
+  string to_denom                = 2;
+  string amount_cap_for_to_denom = 3
+      [(gogoproto.customtype) = "github.com/Finschia/finschia-sdk/types.Int", (gogoproto.nullable) = false];
+  string swap_rate = 4
+      [(gogoproto.customtype) = "github.com/Finschia/finschia-sdk/types.Dec", (gogoproto.nullable) = false];
+}
+
+message SwapStats {
+  option (gogoproto.equal)            = true;
+  option (gogoproto.goproto_stringer) = false;
+
+  int32 swap_count = 1;
```

Comment:
Is this the count of all swaps regardless of the type of swap registered?

Reply:
This counts registered swaps only.